axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -1322,6 +1322,135 @@
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "Traits": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.traits",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "GasImpl": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gas-impl",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "version": "1.3.0"
+        },
+        "GatewayStorage": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gateway-storage",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1"
+        },
+        "GasService": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gas-service",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "gasCollector": "ST3ASWHNH5G542F1V58CQAND5GFN4ZDGXAJHC2VR0"
+        },
+        "GatewayImpl": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gateway-impl",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "version": "1.3.0"
+        },
+        "Gateway": {
+          "address": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1.gateway",
+          "deployer": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "signers": {
+            "weightedSigners": [
+              {
+                "signer": "0x026e4a6fc3a6988c4cd7d3bc02e07bac8b72a9f5342d92f42161e7b6e57dd47e18",
+                "weight": 1
+              },
+              {
+                "signer": "0x02d19c406d763c98d98554c980ae03543b936aad0c3f1289a367a0c2aafb71e8c1",
+                "weight": 1
+              },
+              {
+                "signer": "0x03ea531f69879b3b15b6e3fe262250d5ceca6217e03e4def6919d4bdce3a7ec389",
+                "weight": 1
+              }
+            ],
+            "threshold": 2,
+            "nonce": "0x000000000000000000000000000000000000000000000000000000000038d32c"
+          },
+          "claritySigners": "0c00000003056e6f6e63650200000020000000000000000000000000000000000000000000000000000000000038d32c077369676e6572730b000000030c00000002067369676e65720200000021026e4a6fc3a6988c4cd7d3bc02e07bac8b72a9f5342d92f42161e7b6e57dd47e180677656967687401000000000000000000000000000000010c00000002067369676e6572020000002102d19c406d763c98d98554c980ae03543b936aad0c3f1289a367a0c2aafb71e8c10677656967687401000000000000000000000000000000010c00000002067369676e6572020000002103ea531f69879b3b15b6e3fe262250d5ceca6217e03e4def6919d4bdce3a7ec389067765696768740100000000000000000000000000000001097468726573686f6c640100000000000000000000000000000002",
+          "owner": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "operator": "STX1M8AD16Y5P606QK6FFXS25BQ3R03TWDFDW7J1",
+          "domainSeparator": "0x9f3f816350e47038614b888448ffbcb9f520b2bcdeeff15cec6c08c3c61c4f20",
+          "minimumRotationDelay": 0,
+          "previousSignerRetention": 15
+        },
+        "Governance": {
```

Comment:
governance contract isn't needed, can we remove it for now instead of needing to maintain it?